apps/dashboard/app/helpers/files_helper.rb

```diff
@@ -4,6 +4,23 @@ def sort_by_created_at(transfers)
     transfers.sort_by { |t| - t.created_at }
   end
 
+  def shell_links
+    html = ""
+
+    Configuration.login_clusters.each{ |cluster|
+      link = OodAppLink.new(
+        title: cluster.title,
+        url: OodAppkit::Urls::Shell.new(base_url: '/').url(host: cluster.title.downcase),
+        icon_uri: 'fas://terminal',
+        new_tab: true
+      ).url
+
+      html.concat(link)
+    }
+
+    html.html_safe
+  end
```

Comment:
Inline this code into _shell.html.erb. Also, we need to create a bunch of `<li><a href=...` but above just concats the URLs themselves.